lib/elixir/test/elixir/float_test.exs

```diff
@@ -5,6 +5,17 @@ defmodule FloatTest do
 
   doctest Float
 
+  # TODO remove and replace by assert once we require Erlang/OTP 27+
+  # We can't easily distinguish between -0.0 and +0.0 on previous version
+  defmacrop float_assert({:===, _, [left, right]}) do
```

Comment:
I went with this approach because it gives us:
- useful assertions
- useful error messages on failure
- can easily be removed since it is not here to stay